agent/api/task_windows.go

```diff
@@ -62,7 +62,12 @@ func getCanonicalPath(path string) string {
 // passed to Docker API.
 func (task *Task) platformHostConfigOverride(hostConfig *docker.HostConfig) error {
 	task.overrideDefaultMemorySwappiness(hostConfig)
-	hostConfig.CPUPercent = hostConfig.CPUShares / int64(cpuShareScaleFactor)
+	// Convert the CPUShares to CPUPercent
+	hostConfig.CPUPercent = hostConfig.CPUShares * 100 / int64(cpuShareScaleFactor)
```

Comment:
What's the 100 here?  Please make a named constant expressing what this is.

Reply:
`CPUPercent` is the integer of percentage, this is the conversion: 25% -> 25